openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx

```diff
@@ -153,7 +153,7 @@ const TagsV1 = ({
         {...tagProps}>
         {/* Wrap only content to avoid redirect on closeable icons  */}
         <Link
-          className="no-underline h-full w-full"
+          className="no-underline h-full w-fill-available"
```

Comment:
Can you check we have enough CSS to support cross-browser?
https://caniuse.com/?search=fill-available